### PR TITLE
Unique email username

### PR DESCRIPTION
**Pull request by CNichole726**
Updated UserController to handle checking for unique email and username.

### CODE REVIEW

**Review by lfallert — APPROVED** [review]
Feature working as intended. I created a user, and then attempted to make a second user with the same username and email. I was not able to do this because alert messages popped up to let me know those already existed. Good job.

File: src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java [hunk 1]
```diff
@@ -7,4 +7,5 @@
 @Repository
 public interface UserRepository extends CrudRepository<User, Integer> {
     User findByUsername(String username);
```
Comment (mertzjl91) [inline src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java]: Could we use these in the confirm username/email functions?

Comment (CNichole726) [reply src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java]: went ahead and removed the findByEmail, it was from something else I was trying to work on previously

**Review by mertzjl91 — APPROVED** [review]

**Review by aeydie — APPROVED** [review]
functioning as it should!

**Pull request merged**